myprog.c

```diff
@@ -1,7 +1,9 @@
+// Declaration Of Header Files 
 #include<stdio.h>
 
-int main() {
-  printf("Hello");
-  // print "hello" 
-  return 0;
+// Here Programme Starts
+int main() 
+{
+  printf("Hello"); // print "hello" 
+  return 0;  // Programme is not returning anything so 0
```

Comment:
It's not because program is not returning anything
0 means program executed with no issues
1 means something went wrong etc
read more here [https://shapeshed.com/unix-exit-codes/](https://shapeshed.com/unix-exit-codes/)